### PR TITLE
feat(app): Add and tweak several CP features, add updated body svg paths, update color scheme

### PR DESCRIPTION
**Pull request by mansnym**
Lots of different stuff happening here, check separate commits for details

### CODE REVIEW

**Comment by ernum** [comment]
Is this ready for review?

**Comment by mansnym** [comment]
yes please :)

File: src/components/BodySection.js [hunk 1]
```diff
@@ -87,9 +87,18 @@ export default function BodySection(props) {
           <path d={frontMuscles.body} />
           <path d={frontMuscles.body} transform={mirror} />
         </g>
+        <g
+          className="frontFace"
+          id="FrontFace"
+          stroke="black"
+          fill="white"
```
Comment (ernum) [inline src/components/BodySection.js:94]: Added a strokeWidth of 2 which I think helps the body pop a bit. Think we should change it.

image

Comment (ernum) [reply src/components/BodySection.js:94]: Can push the changes if you like it. Otherwise PR looks good.

Comment (mansnym) [reply src/components/BodySection.js:94]: Yes please!

File: src/components/CirclePacking.js [hunk 2]
```diff
@@ -1,480 +1,641 @@
...
+      .attr('font-size', fontSize)
+      .attr('font-weight', 'bold')
+      .attr('font-variant', 'small-caps')
+      .attr('text-decoration', 'underline')
+      .attr("pointer-events", "none")
+  }
+
+  function buttonSetup() {
+    createExplainText("Sorting filters", 90, 12);
+    createExplainText("Circle size mapping", 265, 12)
```
[the remark this answers is not in the document]
Comment (mansnym) [reply src/components/CirclePacking.js:433]: I'll change and push

**Pull request merged**